src/css/textarea.less

```diff
@@ -10,12 +10,17 @@
 
   .mq-textarea *, .mq-selectable {
     .user-select(text);
-
     position: absolute; // the only way to hide the textarea *and* the
     clip: rect(1em 1em 1em 1em); // blinking insertion point in IE
-
     resize: none; // hotfix: https://code.google.com/p/chromium/issues/detail?id=355199#c1
+  }
+
+  //remove system cursor on iPad
+  .mq-textarea textarea:focus {
+    text-indent: -9999em;
+  }
 
+  .mq-selectable {
     width: 1px; // don't "stick out" invisibly from a math field,
     height: 1px; // can affect ancestor's .scroll{Width,Height}
```

Comment:
I'm able to repro the following on prod right now:
1. open the Settings popover
2. focus one of the "Step:" math fields
3. scroll rightward
![image](https://cloud.githubusercontent.com/assets/225809/8808479/b706cf06-2f97-11e5-93bf-b2ebf34710b6.png)
(One of the DCG unit tests used to fail with something like the above screenshot; but Jason said you guys completely changed your testing system a while back, to a dedicated Windows VM or something, right? I think it was before that, I'd guess that's why your change didn't cause the same test to fail in the same way. Anyway, we should add a test specifically for this.)

The default size of a textarea's is a lot bigger than 13px by 13px, in Chrome it's 198px by 32px as you can see if you click the "Show Textareas" button in the top right corner of the visual tests page (<http://localhost:9292/test/visual.html>); but it's not the size of the math field that matters, it's the size of the smallest ancestor with `overflow-y` that's not `visible` that will see the mysterious increase in `scrollWidth`, such as in this case, [the Settings popover](https://github.com/desmosinc/knox/blob/893797f42f8ae72550d66fa4f301aae692254e25/frontend/desmos/public/assets/styl/popovers.styl#L26-L41) (by the way, we set `.dcg-popover .dcg-popover-interior` to `overflow: auto` in 2 different places).